doc/classes/ImageTextureLayered.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ImageTextureLayered" inherits="TextureLayered" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		Base class for texture types which contain the data of multiple [ImageTexture]s.
 	</brief_description>
 	<description>
+		[ImageTextureLayered] is a texture array. It is internally used to store [LightmapGIData], but can also be used in custom shaders. See also [ImageTexture3D].
```

Comment:
``ImageTextureLayered`` is an abstract class. It can't be used directly This description appears to be for ``Texture2DArray`` which inherits from ``ImageTextureLayered``